src/options.ts

```diff
@@ -21,6 +21,8 @@ $(()=>{
     });
 
     $('#save').on('click', doSave);
+    $('#cancel').on('click', closeOptionDialog);
```

Comment:
The cancel button doesn't work on Microsoft Edge (`window.close()` does nothing). I couldn't find an alternative for Edge. The only "solutions" I could think of is:
```suggestion
    if(navigator.userAgent.indexOf('Edg/') !== -1) // Edge doesn't support closing
        $('#cancel').remove();
    else
        $('#cancel').on('click', closeOptionDialog);
```

Reply:
I was unaware that we support other browsers than Chrome. I just tried it and the settings page does not look very good on Edge. Let me see what I can come up with.

Also, should `You can change shortcuts using the Extensions > Keyboard Shortcuts option in Chrome` be changed into `You can change shortcuts using the Extensions > Keyboard Shortcuts options`?

Reply:
The readme contains the name "EdgeKeePass" 😉
It already looks better on Edge than before. The problem is that the options dialog in Edge has a fixed width of 400px.

Yes. It would make sense to change this text. Good catch.

Reply:
I played around for a bit but I couldn't find a better solution. I also took the opportunity to fix the `host` and `port` inputs appearing on two different lines in Edge because of the smaller dialog. I think it looks acceptable now:

<details>
<summary>Screenshot</summary>

![Settings on the Edge browser](https://user-images.githubusercontent.com/6966049/114611972-d872c500-9ca1-11eb-9625-3ef76aef0cb5.png)
</details>